### PR TITLE
remove call to "bash -c"

### PR DESCRIPTION
**Pull request by ozdanborne**

### CODE REVIEW

File: test_utils/utils.go [hunk 1]
```diff
@@ -145,21 +155,30 @@ func CreateContainer(netconf string, k8sName string, ip string) (container_id, n
...
-		k8s_env = fmt.Sprintf("CNI_ARGS=\"K8S_POD_NAME=%s;K8S_POD_NAMESPACE=test;K8S_POD_INFRA_CONTAINER_ID=whatever\"", k8sName)
+		cniArgs := fmt.Sprintf("CNI_ARGS=K8S_POD_NAME=%s;K8S_POD_NAMESPACE=test;K8S_POD_INFRA_CONTAINER_ID=whatever", k8sName)
 
 		// Append IP=<ip> to CNI_ARGS only if it's not an empty string.
 		if ip != "" {
-			k8s_env = fmt.Sprintf("%s;IP=%s\"", strings.TrimRight(k8s_env, "\""), ip)
+			cniArgs = cniArgs + fmt.Sprintf(";IP=%s", ip)
 		}
+
+		cni_env = append(cni_env, cniArgs)
```
Comment (tomdee) [inline test_utils/utils.go]: @djosborne How come cni_env is an array of strings, but cniArgs is just a semicolon separated string? Does this even work appending them like this?

Comment (djosborne) [inline test_utils/utils.go]: CNI_ARGS is a nested variable. It's value is a semicolon delimited string. e.g., you end up with this as your struct:
```
{
"CNI_CONTAINERID=a",
"CNI_NETNS=b",
"CNI_ARGS=IP=A;FOO=BAR"
}
```

**Comment by tomdee** [comment]
OK, please rebase, fix the conflicts and squash, then I can merge.

**Comment by caseydavenport** [comment]
@ozdanborne is this still something you're working on?

If so, could you please update the description to indicate the purpose of the PR?

**Pull request closed without merging**